starwars/src/components/characterlist.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Characters from './characters';
+
+
+export default function CharacterList(props){
+    return(
+        <div className='characterlist'>
+
+        <div>
+        {props.starwarsChars.map(Character => (
+            <Characters Characters={Character} />
```

Comment:
I think the naming here is a little weird, but you can name things however you want